secondPart/1_videos.js

```diff
@@ -0,0 +1,37 @@
+const convertToJSON = (arr) => {
+    if (Array.isArray(arr)) {
+        const collection = {}
+        arr.forEach(video => {
+            collection[video.id] = video.title
+        })
+        return JSON.stringify(collection);
```

Comment:
actually it wasn't in task, but okay)

Reply:
I thought that this it the "tricky" part where I should understand by myself that the result should be JSON because of the double quotation marks. I will remove this logic.